guides/05_tabular_data_science_and_plots/using_gradio_with_bigquery.md

```diff
@@ -0,0 +1,110 @@
+# Using Gradio with Google BigQuery
```

Comment:
Don't like this title, what about something users who don't know about Gradio would search for? Like, "Creating a Dashboard from Google BigQuery", or "Using Google BigQuery from Python"?

Reply:
Title is most important spot for key words in SEO.

Reply:
Key words you should incorporate in the blog post (in section titles, summary, and potentially the title itself): 
* big query (with space)
* bigquery pricing
* google cloud console   
* google cloud platform
* google dataflow

Reply:
Good, high authority related pages to link to throughout the guide: 
* Chapter 1. What Is Google BigQuery? - O'Reilly
https://www.oreilly.com/library/view/google-bigquery-the/9781492044451/ch01.html
* BigQuery Enterprise Data Warehouse - Google Cloud
https://cloud.google.com/bigquery
* google-cloud-bigquery - PyPI
https://pypi.org/project/google-cloud-bigquery/